src/bridge/trx/index.ts

```diff
@@ -7,9 +7,11 @@
 import { TronWeb } from "tronweb-typings";
```

Comment:
Don't use old typings from an unknown package, it confuses more than helps

Reply:
I agree. I removed `tronweb-typings`